src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -476,6 +476,23 @@ public void testGetUriByFilenameFindsResourceRelativeToRootClasspath() throws Ex
                 content, startsWith("good"));
     }
 
+    @Test
+    public void testGetUriByFilenameFindsAbsoluteResourceOnClasspathPrefix() throws Exception {
+        final String filename = CommonUtil.CLASSPATH_URL_PREFIX
+            + "/" + getPackageLocation()
+            + "/InputCommonUtilTest_empty_checks.xml";
+        final URI uri = CommonUtil.getUriByFilename(filename);
+        assertThat("URI is null for: " + filename, uri, is(not(nullValue())));
```

Comment:
We need at least one full functional test as we do for Checks.
To make sure it works.

Validation for not null is ok, but can we make sure content in URI is loadable XML at least. Not null object doesnot mean it is valid object.

Reply:
Done.

Reply:
not done, please recheck.

Reply:
Line 501 doesn't satisfy the config loading?

Reply:
please merge test at 501 to this test.
this test does not make sense and completely covered by test at 501

Reply:
Done.